Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
Suggestion. `Ubuntu/trusty64` is a little outdated. I have not tried 18.04 yet, but 16.04 LTE works fine with vagrant. Thus, shall we gradually move on a new version of the OS?
`config.vm.box = "ubuntu/xenial64"
`

Reply:
I have tested with provisioning of trusty and xenial VMs and discovered that it takes slightly longer time for xenial (although the box is slightly smaller).  Also there are 10 times more downloads of Trusty at Hashicorp. I believe that more users means more developers and more stable releases. Trusty has LTS until April 2019. Let's see then...